topsort-api-v2.yml

```diff
@@ -337,14 +259,14 @@ components:
             Zero-based, so the product with rank 0 won the auction and had the highest bid.
             In an auction response, the winners array is sorted so rank will match the entry's index"
           minimum: 0
-        winnerType:
+        type:
           type: string
           description: The type of the winning bid.
           enum:
             - product
             - brand
             - vendor
-        winnerId:
+        id:
           type: string
           description: The marketplace's ID of the winning entity, depending on the campaign.
           example: b_Mfk15
```

Comment:
In the assetUrl, please use the right format (`uri`) and change the wording to clarify that this will be served by a CDN controlled by topsort.